test-e2e/app/Tests/MatrixPage/MatrixPageFilters.e2e-spec.ts

```diff
@@ -65,8 +65,7 @@ describe('Matrix Page: Filters', () => {
   it('Show all countries', async () => {
     const COUNTRY = 'Sweden'; //TODO: Remowe hardcode
     waitForVisible(MatrixPage.familyLink.get(0));
-    const totalCountriesBefore = await MatrixPage.familyLink.count();
-
+    const totalCountriesBefore = await MatrixPage.familyLink.count() - 8;
```

Comment:
This is a temporary workaround, this behavior needs to be fixed on the dev side.